.devcontainer/scripts/install_go_dev_tools.sh

```diff
@@ -0,0 +1,9 @@
+#!/usr/bin/env bash
+##
+## This script installs all the tools needed for Go Development
+##
+(
+    go install github.com/fzipp/gocyclo@latest
+    go install github.com/fzipp/gocyclo/cmd/gocyclo@latest
+    curl -sSfL https://raw.githubusercontent.com/golangci/golangci-lint/master/install.sh | sh -s -- -b "$(go env GOPATH)/bin" v1.41.1
```

Comment:
I'm not sure the golangci-lint version needs to be pinned. It's not pinned in the PR checks https://github.com/get-woke/woke/blob/main/.github/workflows/go.yml#L84

Reply:
fixed, thanks!